packages/@uppy/aws-s3/src/index.ts

```diff
@@ -284,10 +284,8 @@ const defaultOptions = {
   allowedMetaFields: true,
   limit: 6,
   getTemporarySecurityCredentials: false as any,
-  // eslint-disable-next-line no-bitwise
   shouldUseMultipart: ((file: UppyFile<any, any>) =>
-    // eslint-disable-next-line no-bitwise
-    (file.size! >> 10) >> 10 > 100) as any as true,
+    (file.size || 0) > 100 * 1024 * 1024) as any as true,
```

Comment:
Don't think we need this anymore?
```suggestion
    (file.size || 0) > 100 * 1024 * 1024),
```

Reply:
Yeah, probably, but I'm not too familiar with this repo, so I couldn't determine the correct types.

If completely removed, it shows this error in `constructor`:

![Screenshot 2025-01-20 at 13 05 44](https://github.com/user-attachments/assets/4a873a4d-7650-4801-8a5c-12fbc426d5e2)